### ISSUE TITLE
Remove evaluations from test data and remove evaluation html files

### ISSUE THREAD

**Issue by kanghj** [TEAMMATES/repo#3386]
After #3370, some of the test json still contain evaluations.

**Comment by MrFool** [comment]
There are some unused evaluation html files that seems to go unused as well, separate issue for that or add along in here?

**Comment by kanghj** [comment]
Oh yup, I'm doing those here too
